lib/vmdb/settings.rb

```diff
@@ -6,9 +6,13 @@
 module Vmdb
   class Settings
     PASSWORD_FIELDS = %i(bind_pwd password amazon_secret).to_set.freeze
+    KNOCKOUT_STRING = "--!".freeze
```

Comment:
Let's pick a character different than `!` because it makes YAML a little weird in certain cases (and forces quoting I believe)